app/src/main/java/rs/urosdragojevic/emotionsters_android/MainActivity.java

```diff
@@ -33,25 +42,46 @@ protected void onCreate(Bundle savedInstanceState) {
         Call<Homescreen> call = jsonPlaceHolderApi.getHomescreen();
 
         call.enqueue(new Callback<Homescreen>() {
+            @SuppressLint("SetTextI18n")
             @Override
-            public void onResponse(Call<Homescreen> call, Response<Homescreen> response) {
+            public void onResponse(@NonNull Call<Homescreen> call, @NonNull Response<Homescreen> response) {
                 if (!response.isSuccessful()) {
                     textView.setText("Code: " + response.code());
                     textView1.setText("Code: " + response.code());
                     return;
                 }
                 Homescreen homescreen = response.body();
-
-                    textView.setText(homescreen.getText());
-                    textView1.setText(homescreen.getText1());
-
+                textView.setText(homescreen.getText());
+                textView1.setText(homescreen.getText1());
             }
+
             @Override
-            public void onFailure(Call<Homescreen> call, Throwable t) {
+            public void onFailure(@NonNull Call<Homescreen> call, @NonNull Throwable t) {
                 textView.setText(t.getMessage());
                 textView1.setText(t.getMessage());
-
             }
         });
     }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuInflater menuInflater = getMenuInflater();
+        menuInflater.inflate(R.menu.menu, menu);
+        SearchView searchView = (SearchView) menu.findItem(R.id.search).getActionView();
+        searchView.setQueryHint("Search here...");
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
+        int itemId = item.getItemId();
+        if (itemId == R.id.account) {
+            Toast.makeText(this, "account is pressed", Toast.LENGTH_SHORT).show();
```

Comment:
Maybe not show these toasts if we do not have any implmentation for the menu yet. Just have it do nothing.

Reply:
Done